scala/README.md

```diff
@@ -382,44 +439,48 @@ It is definitely recommended to read the full Twitter's "[Effective Scala]" guid
 
     1. Instead of `c exists (_ == 0)`, use `c contains 0`.
 
-0. Do not import `scala.collection.mutable._` or even a single mutable collection directly. Instead, import the `mutable` package and use it explicitly as a namespace prefix to denote mutability:
+    This is not only a matter of style: each of these recommendations is algorithmically more efficient that the naive approach.
```

Comment:
"is" is a strong assertion. For many collections libraries hand us, `c.length == 0` and `c.isEmpty` will likely have the same running time. And with Scala 2.12, the overhead of `c exists (_ == 0)` vs `contains` is  The only immediate exception I can think of is that Slick sometimes gives `List`. Can I sell you on `each of these -> in some cases, these recommendations are`?

also, just because I was curious how `contains` is implemented... https://github.com/scala/scala/blob/2.12.x/src/library/scala/collection/SeqLike.scala#L401 :)